src/styles/notifications.less

```diff
@@ -263,4 +263,154 @@ ux-alert {
         background-color: @alert-dark-bg;
         border: 1px solid @alert-dark-border;
     }
+}
+
+/*
+ * Badge
+ */
+
+.ux-badge-container {
+    position: relative;
+
+    .ux-badge {
+        display: block;
+        position: absolute;
+        text-align: center;
+        background-color: #000;
+        color: #fff;
+        box-sizing: border-box;
+        padding: 0;
+        margin: 0;
+        text-transform: none !important;
+        font-family: @font-family !important;
```

Comment:
Please don't use `!important` - especially here, since we use a different font in the Micro Focus theme.